diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-2.0.0/root/etc/e-smith/events/actions/ingo_upgrade mezzanine_patched_e-smith-ingo-2.0.0/root/etc/e-smith/events/actions/ingo_upgrade
--- e-smith-ingo-2.0.0/root/etc/e-smith/events/actions/ingo_upgrade	2010-01-23 22:22:41.000000000 -0600
+++ mezzanine_patched_e-smith-ingo-2.0.0/root/etc/e-smith/events/actions/ingo_upgrade	2010-01-23 18:47:00.000000000 -0600
@@ -47,6 +47,10 @@
 # These are all safe to run multiple times
 
 my @statements = (
+    "ALTER TABLE ingo_shares CHANGE share_flags share_flags SMALLINT DEFAULT 0 NOT NULL",
+    "ALTER TABLE ingo_shares CHANGE perm_creator perm_creator SMALLINT DEFAULT 0 NOT NULL",
+    "ALTER TABLE ingo_shares CHANGE perm_default perm_default SMALLINT DEFAULT 0 NOT NULL",
+    "ALTER TABLE ingo_shares CHANGE perm_guest perm_guest SMALLINT DEFAULT 0 NOT NULL",
     "ALTER TABLE ingo_shares_groups CHANGE group_uid group_uid VARCHAR(255)",
 );
 
diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-2.0.0/root/home/httpd/html/horde/smeserver/ingo.sql mezzanine_patched_e-smith-ingo-2.0.0/root/home/httpd/html/horde/smeserver/ingo.sql
--- e-smith-ingo-2.0.0/root/home/httpd/html/horde/smeserver/ingo.sql	2010-01-23 22:22:41.000000000 -0600
+++ mezzanine_patched_e-smith-ingo-2.0.0/root/home/httpd/html/horde/smeserver/ingo.sql	2009-12-28 10:44:11.000000000 -0600
@@ -1,4 +1,4 @@
--- $Horde: ingo/scripts/sql/ingo.sql,v 1.6.2.4 2008/04/29 19:33:21 chuck Exp $
+-- $Horde: ingo/scripts/sql/ingo.sql,v 1.6.2.8 2009-10-20 21:44:32 jan Exp $
 
 CREATE TABLE IF NOT EXISTS ingo_rules (
     rule_id INT NOT NULL,
@@ -59,10 +59,10 @@
     share_id INT NOT NULL,
     share_name VARCHAR(255) NOT NULL,
     share_owner VARCHAR(255) NOT NULL,
-    share_flags SMALLINT NOT NULL DEFAULT 0,
-    perm_creator SMALLINT NOT NULL DEFAULT 0,
-    perm_default SMALLINT NOT NULL DEFAULT 0,
-    perm_guest SMALLINT NOT NULL DEFAULT 0,
+    share_flags SMALLINT DEFAULT 0 NOT NULL,
+    perm_creator SMALLINT DEFAULT 0 NOT NULL,
+    perm_default SMALLINT DEFAULT 0 NOT NULL,
+    perm_guest SMALLINT DEFAULT 0 NOT NULL,
     attribute_name VARCHAR(255) NOT NULL,
     attribute_desc VARCHAR(255),
     PRIMARY KEY (share_id)
