diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/130AppRegistryIngo mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/130AppRegistryIngo
--- e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/130AppRegistryIngo	1969-12-31 18:00:00.000000000 -0600
+++ mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/130AppRegistryIngo	2006-10-05 21:05:08.000000000 -0500
@@ -0,0 +1,10 @@
+// 130AppRegistryIngo
+$this->applications['ingo'] = array(
+    'fileroot' => dirname(__FILE__) . '/../ingo',
+    'webroot' => $this->applications['horde']['webroot'] . '/ingo',
+    'name' => _("Filters"),
+    'status' => 'active',
+    'provides' => array('mail/blacklistFrom', 'mail/showBlacklist', 'mail/whitelistFrom', 'mail/showWhitelist', 'mail/applyFilters', 'mail/canApplyFilters', 'mail/showFilters'),
+    'menu_parent' => 'imp'
+);
+
