--- e-smith-ingo-1.2/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/prefs.php/100header.1.2.1	2008-05-24 18:05:39.000000000 -0500
+++ e-smith-ingo-1.2/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/prefs.php/100header	2008-09-16 22:00:36.000000000 -0500
@@ -1,6 +1,6 @@
 //100header
 /**
- * $Horde: ingo/config/prefs.php.dist,v 1.30.10.2 2008/05/09 22:22:59 jan Exp $
+ * $Horde: ingo/config/prefs.php.dist,v 1.30.10.3 2008/06/15 21:14:38 chuck Exp $
  *
  * See horde/config/prefs.php for documentation on the structure of this file.
  */
--- e-smith-ingo-1.2/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/prefs.php/120storage.1.2.1	2005-08-31 19:54:50.000000000 -0500
+++ e-smith-ingo-1.2/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/prefs.php/120storage	2008-09-16 22:02:42.000000000 -0500
@@ -1,5 +1,4 @@
 //120storage
 // Make sure that INGO_STORAGE_* constants are defined.
-@define('INGO_BASE', dirname(__FILE__) . '/..');
-require_once INGO_BASE . '/lib/Storage.php';
+require_once dirname(__FILE__) . '/../lib/Storage.php';
 
