diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/100header mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/100header
--- e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/100header	2005-09-01 12:41:12.000000000 -0500
+++ mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/100header	2006-09-14 07:20:55.000000000 -0500
@@ -1,6 +1,6 @@
 //100header
 /**
- * $Horde: ingo/config/backends.php.dist,v 1.20 2004/11/09 10:35:37 jan Exp $
+ * $Horde: ingo/config/backends.php.dist,v 1.20.8.4 2006/02/02 22:57:31 jan Exp $
  *
  * Ingo works purely on a preferred mechanism for server selection. There are
  * a number of properties that you can set for each backend:
@@ -27,6 +27,7 @@
  * script:       The type of Ingo_Script driver this server uses.
  *               Valid options:
  *                   'imap'      --  IMAP client side filtering
+ *                   'maildrop'  --  Maildrop scripts
  *                   'procmail'  --  Procmail scripts
  *                   'sieve'     --  Sieve scripts
  *
diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/130maildrop mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/130maildrop
--- e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/130maildrop	1969-12-31 18:00:00.000000000 -0600
+++ mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/130maildrop	2006-09-14 07:26:17.000000000 -0500
@@ -0,0 +1,43 @@
+//130maildrop
+/* Maildrop Example */
+//$backends['maildrop'] = array(
+//    'driver' => 'vfs',
+//    'preferred' => 'example.com',
+//    'hordeauth' => true,
+//    'params' => array(
+//        // Hostname of the VFS server
+//        'hostspec' => 'ftp.example.com',
+//        // Name of the maildrop config file to write
+//      'filename' => '.mailfilter',
+//        // Port of the VFS server  
+//        'port' => 21,
+//        // The VFS driver to use
+//        'vfstype' => 'ftp',
+//        // The VFS username to use, defaults to current user
+//        // 'username' => 'user',
+//        // The VFS password to use, defaults to current user's password
+//        // 'password' => 'secret',
+//        // The path to the .mailfilter filter file, defaults to the current
+//        // user's home directory.
+//        // You can use the following variables:
+//        //   %u = name of the current user
+//        //   %U = the 'username' from above
+//        // Example:
+//        //   '/data/maildrop/filters/%u/'
+//        //   This would be translated into:
+//        //   '/data/maildrop/filters/<logged_in_username>/.mailfilter'
+//        // 'vfs_path' => '/path/to/maildrop/',
+//    ),
+//    'script' => 'maildrop',
+//    'scriptparams' => array(
+//        // What path style does the IMAP server use ['mbox'|'maildir']?
+//        'path_style' => 'mbox',
+//        // An array of variables to append to every generated script.
+//        // Use if you need to set up specific environment variables.
+//        'variables' => array(
+//            // Example for the $PATH variable
+//            // 'PATH' => '/usr/bin'
+//        )
+//    )
+//);
+
diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/140procmail mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/140procmail
--- e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/140procmail	2005-09-01 14:47:19.000000000 -0500
+++ mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/140procmail	2006-09-14 21:53:22.000000000 -0500
@@ -4,26 +4,42 @@
 //    'driver' => 'vfs',
 //    'preferred' => 'example.com',
 //    'hordeauth' => true,
-//    'params' => array(
+//    'params' => array(  
 //        // Hostname of the VFS server
 //        'hostspec' => 'ftp.example.com',
 //        // Name of the procmail config file to write
-//        'procmailrc' => '.procmailrc',
-//        // Port of the VFS server
+//        'filename' => '.procmailrc',
+//        // Port of the VFS server   
 //        'port' => 21,
 //        // The VFS driver to use
 //        'vfstype' => 'ftp'
+//        // The VFS username to use, defaults to current user
+//        // 'username' => 'user',
+//        // The VFS password to use, defaults to current user's password
+//        // 'password' => 'secret',
+//       // The path to the .procmailrc filter file, defaults to the current
+//        // user's home directory.
+//        // You can use the following variables:
+//        //   %u = name of the current user
+//        //   %U = the 'username' from above
+//        // Example:
+//        //   '/data/procmail/filters/%u/'
+//        //   This would be translated into:
+//        //   '/data/procmail/filters/<logged_in_username>/.procmailrc'
+//        // 'vfs_path' => '/path/to/procmail/',
 //    ),
 //    'script' => 'procmail',
 //    'scriptparams' => array(
 //        // What path style does the IMAP server use ['mbox'|'maildir']?
 //        'path_style' => 'mbox',
-//        // An array of variables to append to every generated script.
-//        // Use if you need to set up specific environment variables.
-//        'variables' => array(
+//       // An array of variables to append to every generated script.
+//        // Use if you need to set up specific environment variables. 
+//       'variables' => array(
 //            // Example for the $PATH variable
-//            // 'PATH' => '/usr/bin'
-//        )
+//            // 'PATH' => '/usr/bin',
+//           // Example for the $DEFAULT variable
+//            // 'DEFAULT' => '$HOME/Maildir',
+//       )
 //    )
-//);
+//);   
 
diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/160sieve mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/160sieve
--- e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/160sieve	2005-09-01 14:46:38.000000000 -0500
+++ mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/160sieve	2006-09-14 21:54:30.000000000 -0500
@@ -4,15 +4,20 @@
 //    'driver' => 'timsieved',
 //    'preferred' => 'example.com',
 //    'hordeauth' => true,
-//    'params' => array(
+//    'params' => array(  
 //        // Hostname of the timsieved server
-//        'hostspec' => 'mail.example.com',
+//        'hostspec' => 'mail.example.com',  
 //        // Login type of the server
 //        'logintype' => 'PLAIN',
 //        // Port number of the timsieved server
 //        'port' => 2000,
 //        // Name of the sieve script
-//        'scriptname' => 'ingo'
+//        'scriptname' => 'ingo',
+//        // The following settings can be used to specify an administration
+//        // user to update all users' scripts.
+//        // 'admin' => 'cyrus',
+//        // 'password' => '*****',
+//        // 'username' => Auth::getAuth(),
 //    ),
 //    'script' => 'sieve',
 //    'scriptparams' => array()
diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/170ldapsieve mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/170ldapsieve
--- e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/170ldapsieve	1969-12-31 18:00:00.000000000 -0600
+++ mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/170ldapsieve	2006-09-14 21:57:44.000000000 -0500
@@ -0,0 +1,61 @@
+//170ldapsieve
+/* Sun ONE/JES Example (LDAP/Sieve) */
+//$backends['ldapsieve'] = array(
+//    'driver' => 'ldap',
+//    'preferred' => 'example.com',
+//    'hordeauth' => false,
+//    'params' => array(
+//        //
+//        // Hostname of the ldap server
+//        //
+//        'hostspec' => 'ldap.example.com',
+//        //
+//        // Port number of the timsieved server
+//        //
+//        'port' => 389,
+//        //
+//        // LDAP Protocol Version (default = 2).  3 is required for TLS.
+//        //
+//        'version' => 3,
+//        //
+//        // Whether or not to use TLS.  If using TLS, you MUST configure
+//        // OpenLDAP (either /etc/ldap.conf or /etc/ldap/ldap.conf) with the CA
+//        // certificate which signed the certificate of the server to which you
+//        // are connecting.  e.g.:
+//        //
+//        // TLS_CACERT /usr/share/ca-certificates/mozilla/Equifax_Secure_CA.crt
+//        //
+//       // You MAY have problems if you are using TLS and your server is
+//        // configured to make random referrals, since some OpenLDAP libraries
+//        // appear to check the certificate against the original domain name,
+//       // and not the referred-to domain.  This can be worked around by
+//        // putting the following directive in the ldap.conf:
+//        // 
+//        // TLS_REQCERT never
+//        // 
+//        'tls' => true,
+//        //
+//        // Bind DN (for bind and script distinguished names, %u is replaced
+//        // with username, and %d is replaced with the internet domain
+//        // components (e.g. "dc=example, dc=com") if available).
+//        // 
+//        'bind_dn' => 'cn=ingo, ou=applications, dc=example, dc=com',
+//        //
+//        // Bind password.  If not provided, user's password is used (useful
+//       // when bind_dn contains %u).
+//        // 
+//        'bind_password' => 'secret',
+//        //
+//        // How to find user object.
+//        // 
+//        'script_base' => 'ou=People, dc=example, dc=com',
+//        'script_filter' => '(uid=%u)',
+//        //
+//        // Attribute script is stored in.  Will not touch non-Ingo scripts.
+//        // 
+//        'script_attribute' => 'mailSieveRuleSource'
+//    ),
+//    'script' => 'sieve',
+//    'scriptparams' => array()
+//);
+
