diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/100header mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/100header
--- e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/100header	2006-10-23 22:27:49.000000000 -0500
+++ mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/100header	2006-10-23 22:23:25.000000000 -0500
@@ -1,6 +1,6 @@
 //100header
 /**
- * $Horde: ingo/config/backends.php.dist,v 1.20.8.4 2006/02/02 22:57:31 jan Exp $
+ * $Horde: ingo/config/backends.php.dist,v 1.20.8.6 2006/10/09 15:52:48 jan Exp $
  *
  * Ingo works purely on a preferred mechanism for server selection. There are
  * a number of properties that you can set for each backend:
@@ -10,6 +10,7 @@
  *                   'null'       --  No backend server
  *                   'timsieved'  --  Cyrus timsieved server
  *                   'vfs'        --  Use Horde VFS
+ *                   'ldap'       --  LDAP server
  *
  * preferred:    This is the field that is used to choose which server is
  *               used. The value for this field may be a single string or an
diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/160sieve mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/160sieve
--- e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/160sieve	2006-10-23 22:27:49.000000000 -0500
+++ mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/160sieve	2006-10-23 22:25:28.000000000 -0500
@@ -9,6 +9,8 @@
 //        'hostspec' => 'mail.example.com',  
 //        // Login type of the server
 //        'logintype' => 'PLAIN',
+//        // Enable/disable TLS encryption
+//        'usetls' => true,
 //        // Port number of the timsieved server
 //        'port' => 2000,
 //        // Name of the sieve script
diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/180kolabsieve mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/180kolabsieve
--- e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/180kolabsieve	2005-09-01 14:46:04.000000000 -0500
+++ mezzanine_patched_e-smith-ingo-1.1/root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/backends.php/180kolabsieve	2006-10-23 22:26:53.000000000 -0500
@@ -8,6 +8,7 @@
 //        'params' => array(
 //            'hostspec' => $GLOBALS['conf']['kolab']['imap']['server'],
 //            'logintype' => 'PLAIN',
+//            'usetls' => false,
 //            'port' => $GLOBALS['conf']['kolab']['imap']['sieveport'],
 //            'scriptname' => 'kmail-vacation.siv'
 //        ),
