diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-1.1/createlinks mezzanine_patched_e-smith-ingo-1.1/createlinks
--- e-smith-ingo-1.1/createlinks	2005-08-31 18:37:01.000000000 -0500
+++ mezzanine_patched_e-smith-ingo-1.1/createlinks	2006-09-23 12:15:35.000000000 -0500
@@ -2,6 +2,7 @@
 
 use File::Basename;
 use File::Path;
+use esmith::Build::CreateLinks  qw(:all);
 
 sub safe_touch
 {
@@ -37,3 +38,11 @@
 	    email-update
 	));
 }
+
+foreach (qw(conf.php prefs.php backends.php))
+{
+    safe_symlink("/etc/e-smith/templates-default/template-begin-php",
+        "root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/$_/template-begin");
+    safe_symlink("/etc/e-smith/templates-default/template-end-php",
+        "root/etc/e-smith/templates/home/httpd/html/horde/ingo/config/$_/template-end");
+}
