diff -Nur -x '*.orig' -x '*.rej' e-smith-ingo-1.0.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/85IngoAccess mezzanine_patched_e-smith-ingo-1.0.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/85IngoAccess
--- e-smith-ingo-1.0.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/85IngoAccess	2005-08-31 19:42:52.000000000 -0400
+++ mezzanine_patched_e-smith-ingo-1.0.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/85IngoAccess	2006-06-14 11:46:18.000000000 -0400
@@ -1,16 +1,8 @@
 {
-    use esmith::db;
-    use esmith::config;
+    my $status = $imp{status} || 'disabled';
 
-    local %services;
-    $services{'imp'} = $imp;
-
-    my $status = db_get_prop(\%services, "imp", "status");
-
-    if (defined $status)
+    if ($status eq 'enabled')
     {
-	if ($status eq 'enabled')
-	{
 	    $OUT .= qq(
 # INGO specific access configuration
 
@@ -34,6 +26,5 @@
     deny from all
 </Directory>
 );
-	}
     }
 }
